### PR TITLE
Added new role Partner Manager (PM) and its group.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-6545](https://openedx.atlassian.net/browse/ECOM-6545)

As a Publisher user I need to manage PM (Partner Manager) Role.

**AC:**

- Validate that we have a Partner Manager role
- Validate that we have a group for Partner Manager

screen shot 2017-01-19 at 2 51 51 pm

### CODE REVIEW

**Comment by waheedahmed** [comment]
@tasawernawaz, @awais786 Please review.

File: course_discovery/apps/publisher/api/tests/test_views.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def setUp(self):
 
         self.internal_user_group.user_set.add(self.internal_user)
         self.other_internal_users = []
-        for __ in range(3):
+        for __ in range(4):
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/api/tests/test_views.py]: Can we use `UserFactory().batch()` here instead of `for loop`.

**Review by tasawernawaz — APPROVED** [review]
Just a minor question otherwise LGTM.

**Comment by waheedahmed** [comment]
@bderusha Please review.

**Review by bderusha — APPROVED** [review]

**Pull request merged**